core/tests/protractor_utils/workflow.js

```diff
@@ -136,6 +130,13 @@ var getExplorationPlaytesters = function() {
   return _getExplorationRoles('viewer');
 };
 
+var getNumberOfFeedbackMessages = function() {
```

Comment:
Because the creator can get to the feedback tab only from the creator dashboard(as far as I know). Since the action is not happening in the editor, I decided to keep it in `workflow`.